src/styles/variable.scss

```diff
@@ -0,0 +1,5 @@
+$blockWidth: 166px;
```

Comment:
```suggestion
$blockWidth: 100%;
```

Reply:
this won't correct work

Reply:
Because you need to add box-sizing and padding for the card